personality_insights/watson-personality-insights-readme.md

```diff
@@ -0,0 +1,225 @@
+
+```ruby
+set :pi_endpoint, "http://MY_APPLICATION_ENDPOINT.mybluemix.net/pi/"
+```
+
+
+Add the following code to the sample proxy.
+
+```
+post '/pi/model_text/' do
+   data = JSON.load(request.body)
+
+   url = "#{settings.personality_insights_endpoint}model_text/"
+   body = { :text => data["text"] }
+   headers = { "Content-Type" => "application/json" }
+
+   response = Excon.post(url, :body => body.to_json, :headers => headers)
+   response.body
+end
+
+
+post '/pi/model_twitter/' do
+   data = JSON.load(request.body)
+
+   url = "#{settings.personality_insights_endpoint}model_tweets/#{data["handle"]}"
+   response = Excon.get(url, :headers => headers)
+
+   response.body
+end
+```
+
+Once you have updated the Proxy, restart the Application Builder server for the changes to take effect.
+
+
+#### Building Personality Insights widget #1: Send Text to the Personality Insights Service
+
+The purpose of this widget is to send a single block of text to the Personality Insights service for analysis. The results of that analysis are then displayed in the Application Builder UI.
+
+Once you have logged into the Application Builder Administration tool, follow these steps to create the custom widget and add it to the search results page.
+
+1. In the Application Builder Administration Tool, navigate to the Pages & Widgets -> Book Title page. (If you have different entities in your application, choose one that has a snippet or be prepared to modify the example widget code.)
+2. Create a Custom new widget.
+3. Set the ID of the widget to be `Watson_PI`
+4. Set the title of the widget to be `Watson Personality Insights`
+5. Copy and paste the [code for this widget](/personality_insights/ApplicationBuilder/personality-insights-text-widget.erb) into the Type Specific Configuration.
+6. Click to turn "Asynchronously load content" on.
+7. Save the widget.
+8. Go back to the Book Title page
+9. Add the `Watson_PI` widget to the Book Title page and save the page configuration.
+
+At this point the widget should be fully configured.  To test the widget, navigate to the application and choose a Book Title entity. The example shown in the below image uses the title from "Lug Nuts!" to create the model displayed. 
+
+This simple example illustrates a basic integration. You should choose text that is relevant to a user or entity that you are modeling and you must provide enough text to generate a reliable model.  See the [Personality Insights documentation](http://www.ibm.com/smarterplanet/us/en/ibmwatson/developercloud/personality-insights.html) for more guidance on text selection and recommendations.
+
+![Screen shot of "Watson Personality Insights" widget.](/personality_insights/ApplicationBuilder/presonality-insights-widget.png)
```

Comment:
We should take this opportunity to correct the image filename and this link. "PRES"onality vs "PERS"onality